session.go

```diff
@@ -270,9 +271,11 @@ func (s *session) recvLoop() {
 			}
 
 			c, open := s.getOrCreateStream(id)
-			if !open && !alreadyLoggedReceiveForClosedStream[id] {
-				log.Debugf("Received data for closed stream %d", id)
-				alreadyLoggedReceiveForClosedStream[id] = true
+			if !open {
+				if !alreadyLoggedReceiveForClosedStream[id] {
+					log.Debugf("Received data for closed stream %d", id)
+					alreadyLoggedReceiveForClosedStream[id] = true
+				}
```

Comment:
Yeah I added the same fix (and just closed it) at https://github.com/getlantern/lampshade/pull/25